lib/zendesk_apps_tools/server.rb

```diff
@@ -4,9 +4,20 @@
 module ZendeskAppsTools
   class Server < Sinatra::Base
     set :public_folder, Proc.new {"#{settings.root}/assets"}
+    last_mtime = Time.new(0)
 
     get '/app.js' do
       content_type 'text/javascript'
+
+      if (File.exists? settings.config)
+        curr_mtime = File.stat(settings.config).mtime
+        if (curr_mtime > last_mtime)
+          settings_helper = ZendeskAppsTools::Settings.new
+          settings.parameters = settings_helper.get_settings_from_file settings.config, settings.manifest
+          last_mtime = File.stat(settings.config).mtime
```

Comment:
any reason you're not setting it to the cached mtime (`curr_mtime`) but looking it up again?